backend/btrixcloud/utils.py

```diff
@@ -123,6 +124,15 @@ def slug_from_name(name: str) -> str:
     return slugify(name.replace("'", ""))
 
 
+def validate_slug(slug: str) -> None:
+    """Validate org slug, raise HTTPException if invalid or already exists
```

Comment:
```suggestion
    """Validate org slug, raise HTTPException if invalid
```

This can only check if its invalid.